docs/user-guide/zonal-average.ipynb

```diff
@@ -10,6 +10,21 @@
     "This section demonstrates how to perform Zonal Averaging using UXarray, covering both non-conservative and conservative methods.\n"
```

Comment:
<p>Please see if the <code>matplotlib.axis.XTick</code> artifacts can be avoided. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/1403/discussion/#comment-2520082367'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>